chingu-coders/Voyage2-Turtles-02
# quicklinks delete option and refactor

## Purpose
* Added delete option to links in quicklink
* general refactor of quiclinks code

## Approach/Fixes
* Implements basic quicklink feature
* improves performance and readability of code

## Testing
* Checkout this PR
* open quicklinks and confirm each link has an `x` next to it
* click `x` and confirm link is deleted from view, and from local storage
* test adding a link as well
